swap_meet/clothing.py

```diff
@@ -1,2 +1,8 @@
-class Clothing:
-    pass
\ No newline at end of file
+from .item import Item
+
+class Clothing(Item):
+    def __init__(self, condition = 0, age = 2):
+        super().__init__(category = "Clothing", condition = condition, age = age)
+
+    def __str__(self):
+        return "The finest clothing you could wear."
```

Comment:
👍  Good work in ensuring that all `Clothing` instances have a `category` of `"Clothing"` by passing the value into the parent constructor.